cmd.go

```diff
@@ -495,6 +498,15 @@ func copyFlagSet(cfg config, f *compflag.FlagSet) *compflag.FlagSet {
 	return (*compflag.FlagSet)(cp)
 }
 
+func detectCompletionSupport() bool {
+	if shellPath, ok := os.LookupEnv("SHELL"); ok {
```

Comment:
using `os.Getenv` would be sufficient, and will simplify the code here.

Reply:
👍